src/madpack/madpack.py

```diff
@@ -357,7 +356,7 @@ def _get_rev_num(rev):
         @param rev version text
     """
     try:
-        num = re.findall('[0-9]', rev)
```

Comment:
Seems that rev is from `match_details.group(1)` which only contains numbers. Feel free to ignore the above comments.